coalib/output/ConsoleInteraction.py

```diff
@@ -135,32 +135,37 @@ def acquire_actions_and_apply(console_printer,
     """
     cli_actions = CLI_ACTIONS if cli_actions is None else cli_actions
     failed_actions = set()
-    while True:
-        actions = []
-        for action in cli_actions:
-            if action.is_applicable(result, file_dict, file_diff_dict) is True:
-                actions.append(action)
-
-        if actions == []:
-            return
+    successful_actions = set()
```

Comment:
Yeah, i guess using a set was only to be consistent with the failed_action variable, which is also a set :)
Shouldn't be a problem to use a list instead.